.github/workflows/test.yml

```diff
@@ -36,24 +48,26 @@ jobs:
           flag-name: run-linux
           parallel: true
 
-  build-container:
-    runs-on: ubuntu-latest
-
+  harvest-test-linux:
+    name: Linux harvest tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-18.04, ubuntu-16.04 ]
```

Comment:
Why we run tests only with this 2 versions of ubuntu?

Reply:
https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners

Reply:
And for `ubuntu 20.04` harvesting tests are not ready, right?

Reply:
they don't pass for ubuntu 20.04 because of the pkg plugin